examples/chatgpt/gpt_actions_library/gpt_action_snowflake_direct.ipynb

```diff
@@ -238,16 +238,48 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Before you set up authentication in ChatGPT, please take the following steps in the application."
+    "Before you set up authentication in ChatGPT, please take the following steps in Snowflake.\n",
+    "\n",
+    "* Review the Snowflake documentation on [Network Policies](https://docs.snowflake.com/en/user-guide/network-policies)\n",
+    "* Go to the Snowflake Worksheets\n",
+    "* Create a network rule with the ChatGPT IP egress ranges listed [here](https://platform.openai.com/docs/actions/production/ip-egress-ranges)\n",
+    "* Create a corresponding Network Policy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "yaml"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "CREATE NETWORK RULE chatgpt_network_rule\n",
+    "  MODE = INGRESS\n",
+    "  TYPE = IPV4\n",
+    "  VALUE_LIST = ('23.102.140.112/28',\n",
+    "'13.66.11.96/28',\n",
+    "'104.210.133.240/28',\n",
+    "'20.97.188.144/28',\n",
+    "'20.161.76.48/28',\n",
+    "'52.234.32.208/28',\n",
+    "'52.156.132.32/28',\n",
+    "'40.84.220.192/28',\n",
+    "'23.98.178.64/28',\n",
+    "'40.84.180.128/28');\n",
+    "\n",
+    "CREATE NETWORK POLICY chatgpt_network_policy\n",
+    "  ALLOWED_NETWORK_RULE_LIST = ('chatgpt_network_rule');"
```

Comment:
Is indentation important? Looks a bit unaligned when rendered
<img width="1043" alt="image" src="https://github.com/user-attachments/assets/1c2192d4-2f55-49ef-9132-d25e999ddacb">

Reply:
not important. i just indented more to line it up. not sure what would look best here. maybe all aligned, or all in a single line?